### PR TITLE
Private_LAW_ Add Work type changes for private law service

### PR DESCRIPTION
**Pull request by Karthikarenjith**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/AM-2516

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/functionalTest/resources/features/F-008/S-071.td.json [hunk 1]
```diff
@@ -135,7 +134,7 @@
             "primaryLocation" : "[[ANYTHING_PRESENT]]",
             "jurisdiction" : "PRIVATELAW",
             "region" : "[[ANYTHING_PRESENT]]",
-            "workTypes" : "routine_work"
+            "workTypes" : "[[ANYTHING_PRESENT]]"
```
Comment (sankar-padakula) [inline src/functionalTest/resources/features/F-008/S-071.td.json:137]: we have to add proper worktypes. in consist values only we have to use ANYTHING_PRESENT. So user proper worktype here. and same for remaining places as well

File: src/main/resources/validationrules/privatelaw/privatelaw-admin-mapping.drl [hunk 2]
```diff
@@ -28,7 +28,7 @@ then
    Map<String,JsonNode> attribute = new HashMap<>();
    attribute.put("jurisdiction", JacksonUtils.convertObjectIntoJsonNode("PRIVATELAW"));
    attribute.put("primaryLocation", JacksonUtils.convertObjectIntoJsonNode($cap.getPrimaryLocationId()));
-   attribute.put("workTypes", JacksonUtils.convertObjectIntoJsonNode("routine_work,access_requests,hearing_work,applications"));
+   attribute.put("workTypes", JacksonUtils.convertObjectIntoJsonNode("routine_work,hearing_work applications"));
```
Comment (sankar-padakula) [inline src/main/resources/validationrules/privatelaw/privatelaw-admin-mapping.drl:31]: comma separator is missed between hearing_work and applications as these are two separate work types

File: src/main/resources/validationrules/privatelaw/privatelaw-admin-mapping.drl [hunk 3]
```diff
@@ -59,7 +59,7 @@ then
    attribute.put("jurisdiction", JacksonUtils.convertObjectIntoJsonNode("PRIVATELAW"));
    attribute.put("primaryLocation", JacksonUtils.convertObjectIntoJsonNode($cap.getPrimaryLocationId()));
    attribute.put("region", JacksonUtils.convertObjectIntoJsonNode($cap.getRegionId()));
-
+   attribute.put("workTypes", JacksonUtils.convertObjectIntoJsonNode("routine_work, hearing_work, applications"));
```
Comment (sankar-padakula) [inline src/main/resources/validationrules/privatelaw/privatelaw-admin-mapping.drl:62]: don't give space between comma and work type. follow the same format,

File: src/main/resources/validationrules/privatelaw/privatelaw-admin-mapping.drl [hunk 4]
```diff
@@ -145,7 +144,7 @@ then
    attribute.put("jurisdiction", JacksonUtils.convertObjectIntoJsonNode("PRIVATELAW"));
    attribute.put("primaryLocation", JacksonUtils.convertObjectIntoJsonNode($cap.getPrimaryLocationId()));
    attribute.put("region", JacksonUtils.convertObjectIntoJsonNode($cap.getRegionId()));
-   attribute.put("workTypes", JacksonUtils.convertObjectIntoJsonNode("routine_work"));
+   attribute.put("workTypes", JacksonUtils.convertObjectIntoJsonNode("access-request"));
```
Comment (sankar-padakula) [inline src/main/resources/validationrules/privatelaw/privatelaw-admin-mapping.drl:147]: this should be under score. not hypen

**Review by sankar-padakula — APPROVED** [review]

**Review by sankar-padakula — APPROVED** [review]

**Review by nitinsahu1 — APPROVED** [review]

**Pull request merged**